frontend/screens/InfluencerConfirmAccount.js

```diff
@@ -0,0 +1,485 @@
+                      {payload.email}
+                    </Text>
+                  </View>
+                </View>
+              </View>
+            </View>
+          </View>
+          <View style={styles.depth1Frame2}>
+            <View style={styles.depth2Frame01}>
+              <View style={styles.depth3Frame02}>
+                <View style={styles.depth5Frame0}>
+                  <Text style={[styles.brandType, styles.emailIdLayout]}>
+                    Password
+                  </Text>
+                </View>
+              </View>
+              <View style={[styles.depth3Frame12, styles.frameLayout1]}>
+                <View style={styles.frameLayout1}>
+                  <View style={styles.frameLayout1}>
+                    <Text style={[styles.sophiagetglocom, styles.emailIdTypo]}>
+                      {payload.password}
+                    </Text>
+                  </View>
+                </View>
+              </View>
+            </View>
+          </View>
+          <View style={styles.depth1Frame4}>
+            <View style={[styles.depth2Frame04, styles.depth2FramePosition]}>
+              <View style={styles.depth3Frame05}>
+                <View style={styles.depth4Frame07}>
+                  <View style={styles.depth5Frame0}>
+                    <Text style={[styles.brandType, styles.emailIdLayout]}>
+                      Username
+                    </Text>
+                  </View>
+                </View>
+                <View style={[styles.depth4Frame11, styles.depth4FrameLayout]}>
+                  <View style={styles.depth5Frame0}>
+                    <Text style={[styles.selectBrandType, styles.emailIdTypo]}>
+                      Create a username
+                    </Text>
+                  </View>
+                </View>
+              </View>
+            </View>
+            <Text
+              style={{
+                fontSize: FontSize.size_base,
+                fontFamily: FontFamily.workSansRegular,
+                textAlign: "right",
+              }}
+            >
+              {payload.userName}
+            </Text>
+          </View>
+          <View style={[styles.depth1Frame7, styles.depth1FrameLayout]}>
+            <TouchableOpacity
+              style={{ width: "100%" }}
+              onPress={registerInfluencer}
+            >
+              <View style={[styles.depth2Frame05, styles.frameBg]}>
+                <View style={[styles.depth3Frame07, styles.frameBg]}>
+                  <View style={styles.depth5Frame0}>
+                    <Text style={[styles.createAccount, styles.emailIdLayout]}>
+                      Create account
+                    </Text>
+                  </View>
+                </View>
+              </View>
+            </TouchableOpacity>
+          </View>
+        </View>
+      </View>
+    </ScrollView>
+  );
+};
+
+export default InfluencerConfirmAccount;
+
+const styles = StyleSheet.create({
```

Comment:
similar here, lets keep the style and js file in a separate module called `influencer`